tests/network/vmi_istio.go

```diff
@@ -114,6 +114,8 @@ var _ = SIGDescribe("[Serial] Istio", func() {
 			virtClient, err = kubecli.GetKubevirtClient()
 			util.PanicOnError(err)
 
+			libnet.SkipWhenClusterNotSupportIpv4(virtClient)
```

Comment:
Shouldn't we check that we are not trying to run this on ipv6 mode ?

Reply:
IPv4 only cluster should run the tests.
IPv6 only cluster should skip the test => that's what this skip achieves.
Dual stack cluster should run the tests using the IPv4 addresses only - the test assumes the primary address is IPv4, it is the old behaviour and not relevant to this PR. Fixed the vmIP retrieval to `vmiIP := libnet.GetVmiPrimaryIpByFamily(vmi, k8sv1.IPv4Protocol)` instead of assuming the primary IP is IPv4.

There are some tests using `ingressGatewayServiceIP = ingressGatewayService.Spec.ClusterIP`. I don't know how this service is defined and whether we can assume this IP is IPv4. Since it is not in the scope of this PR, left it untouched.